misc/ninja_syntax.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/python
+#!/usr/bin/python3
```

Comment:
Not sure, can Python 2 still use the file after this change?

Reply:
```suggestion
#!hint/python
```

This file doesn't use a shebang, it does not need one since it cannot be run as a script. A shebang could in theory make text editors that do not recognize the .py file extension but do sniff shebangs, recognize it as a python formatted file and do the appropriate syntax highlighting. You can make this explicit by using a shebang that is ***guaranteed*** to never, ever, ever work.

On the other hand, you could also just delete this line entirely. Probably any software will just detect the .py file extension...

Reply:
Removed it, I agree the extension should be enough for editors